hub/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN pip3 install --upgrade pip
 
 RUN pip3 --no-cache-dir install jupyterhub requests-futures jupyterhub-dummyauthenticator
 
-RUN pip3 --no-cache-dir install git+https://github.com/jupyterhub/kubespawner.git
+RUN pip3 --no-cache-dir install git+https://github.com/data-8/kubespawner.git@volumes
```

Comment:
We are using the @volumes bracnh of kubespawner for now because this fork has the updated code we need (which is not yet merged into official jupyterhub master)